x-pack/test/security_solution_api_integration/test_suites/entity_analytics/default_license/risk_engine/risk_engine_privileges.ts

```diff
@@ -67,6 +67,8 @@ const ROLES = [
   },
 ];
 
+const FORBIDDEN_STATUS_CDOE = 403;
```

Comment:
```suggestion
const FORBIDDEN_STATUS_CODE = 403;
```

But also: I don't think this value is going to change in the near future, so we're probably okay just using `403` directly in these tests 😉